src/cloud/authorize.js

```diff
@@ -110,17 +97,31 @@ async function checkGCloudConfig(environment, options = {}) {
 async function checkSecrets(environment) {
   const secretsDir = path.resolve('deployment', 'environments', environment, 'secrets');
   if (fs.existsSync(secretsDir)) {
-    const secretFiles = await exec(`ls ${secretsDir}`);
-    if (secretFiles) {
-      console.info(red('---'));
-      console.info(red('---'));
-      console.info(
-        red(
-          `--- Warning: Found files in deployment/environments/${environment}/secrets/ - make sure to remove these!`
-        )
-      );
-      console.info(red('---'));
-      console.info(red('---'));
+    const secretFilesLS = await exec(`ls ${secretsDir}`);
+    const secretFiles = secretFilesLS.split('\n').filter((file) => file.endsWith('.conf'));
+    for (const secretFile of secretFiles) {
+      const secretName = secretFile.slice(0, -5);
+      const secretInfo = await getSecretInfo(secretName);
+      if (!secretInfo) {
+        console.info(
+          red(
+            `Warning: Found secret file deployment/environments/${environment}/secrets/${secretFile} that has not been created on the cluster.`
+          )
+        );
+        let confirmed = await prompt({
+          type: 'confirm',
+          name: 'subcommand',
+          message: `Would you like to create secret "${secretName}" now?`,
+          initial: true,
+        });
+        if (confirmed) await setSecret(environment, secretName);
```

Comment:
very tiny nit but I'm more of a fan of brackets and new line for all `if`s instead of one liners..not sure if we have established a position on this one... whichever we do maybe can add an eslint to enforce

Reply:
Ok, I'm fine either way. As long as there never is a newline without a block.